wdl-lint/Cargo.toml

```diff
@@ -16,6 +16,9 @@ wdl-ast = { path = "../wdl-ast", version = "0.9.0" }
 convert_case = { workspace = true }
 indexmap = { workspace = true }
 rowan = { workspace = true }
+serde_json = { workspace = true }
```

Comment:
Nit: let's make these alphabetical (except for the `wdl-ast` package---we generally use the convention of putting our own, local packages first in the dependencies).